pallets/connectors/src/lib.rs

```diff
@@ -157,9 +149,17 @@ pub mod pallet {
 		type Tokens: Mutate<Self::AccountId>
 			+ Inspect<
 				Self::AccountId,
-				AssetId = <Self as pallet::Config>::CurrencyId,
+				AssetId = CurrencyIdOf<Self>,
 				Balance = <Self as pallet::Config>::Balance,
 			> + Transfer<Self::AccountId>;
+
+		type TrancheToken: TrancheToken<PoolIdOf<Self>, TrancheIdOf<Self>, CurrencyIdOf<Self>>;
```

Comment:
This trait is deprecated. Need to remove it. Sorry for that. Please use `TrancheCurrency`

`type TrancheCurrency: TrancheCurrency<PoolIdOf<Self>, TrancheIOf<Self>> + Into<CurrencyId<Of>;`